instrumentation/classloaders/integration-test/src/test/groovy/AddUrlTest.groovy

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
+import org.apache.commons.io.IOUtils
+import org.apache.commons.lang3.SystemUtils
+
+class AddUrlTest extends AgentInstrumentationSpecification {
+
+  def "should instrument class after it is loaded via addURL"() {
```

Comment:
I think that we have to test that instrumentation was NOT applied before `addURL`.

Reply:
added verification 👍